test/fixtures/restaurants.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  address: MyString
```

Comment:
Again you are pushing unnecessary files. Why you are doing same mistake again and again.
cc: @kalpana19sharma